### ISSUE TITLE
Fatal error thrown when creating a Hero Image paragraph with empty field_link

### ISSUE THREAD

**Issue by bmunslow** [Gizra/drupal-starter#551]
Steps to reproduce:

- Create new Landing Page
- Add Hero Image paragraph
- Fill in mandatory fields (Title and Image) but leave Link field empty
- Save

https://github.com/Gizra/drupal-starter/assets/295192/5fb21980-698c-48e8-9116-d32c27546813

**Comment by bmunslow** [comment]
Error output:

```
The website encountered an unexpected error. Please try again later.

TypeError: Drupal\server_general\Plugin\EntityViewBuilder\ParagraphHeroImage::buildElementHeroImage(): Argument #4 ($button_text) must be of type string, null given, called in /var/www/html/web/modules/custom/server_general/src/Plugin/EntityViewBuilder/ParagraphHeroImage.php on line 47 in Drupal\server_general\Plugin\EntityViewBuilder\ParagraphHeroImage->buildElementHeroImage() (line 80 of modules/custom/server_general/src/ElementTrait.php).
Drupal\server_general\Plugin\EntityViewBuilder\ParagraphHeroImage->buildFull(Array, Object) (Line: 100)
Drupal\pluggable_entity_view_builder\EntityViewBuilderPluginAbstract->build(Array, Object, 'full') (Line: 147)
Drupal\pluggable_entity_view_builder\ParagraphViewBuilder->doBuild(Array) (Line: 66)
Drupal\pluggable_entity_view_builder\ParagraphViewBuilder->doBuildMultiple(Array) (Line: 37)
Drupal\pluggable_entity_view_builder\ParagraphViewBuilder->buildMultiple(Array) (Line: 238)
Drupal\Core\Entity\EntityViewBuilder->build(Array)
```

**Comment by amitaibu** [comment]
@bmunslow Thanks. Do you want to have a shot at writing a PR for this? 😸 We should keep the Link field optional, but fix the WSOD.

**Comment by GizraNotifier** [comment]
The latest merged PR #554 just got deployed successfully to Pantheon [`qa`](https://gizra-drupal-starter-qa.pantheonsite.io) environment
